docs/source/getting_started/installation.md

```diff
@@ -1,68 +1,76 @@
 (target-installation)=
 # Installation
 
-## Create a conda environment
+## Install the package
 
 :::{admonition} Use a conda environment
 :class: note
 We recommend you install movement inside a [conda](conda:)
-or [mamba](mamba:) environment, to avoid dependency conflicts with other packages.
-In the following we assume you have `conda` installed,
+or [mamba](mamba:) environment, to avoid dependency conflicts with other packages. In the following we use `movement-env` as the environment name, but you can choose any name you like.
+We will assume you have `conda` installed,
 but the same commands will also work with `mamba`/`micromamba`.
 :::
 
-First, create and activate an environment with some prerequisites.
-You can call your environment whatever you like, we've used `movement-env`.
 
+::::::{tab-set}
+:::::{tab-item} Users
+::::{tab-set}
+:::{tab-item} Conda
+Create and activate an environment with movement installed.
+```sh
+conda create -n movement-env -c conda-forge movement
+conda activate movement-env
+```
+(Optional) Update an existing installation of movement to the latest version within the same environment:
```

Comment:
We could move the update part outside of both tabs, to reuse the common sentence. Right now it also feels a bit hidden.

I had a go [here](https://github.com/neuroinformatics-unit/movement/pull/264/commits/b96ee86ca028a6484d97df7c186213457efb674a) but again feel free to revert if you disagree. Below how it looks:

<img width="1055" alt="image" src="https://github.com/user-attachments/assets/263ce9c2-4f18-4e90-8f86-85a0cef2e320">